starter-code/bin/seeds.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require('mongoose');
+const Celebrity = require("../models/celebrity")
```

Comment:
This import is incomplete... it will crash your app. 
```suggestion
const Celebrity = require("../models/celebrity.model")
```